backend/src/graphql/model/UserAdmin.ts

```diff
@@ -14,7 +16,12 @@ export class UserAdmin {
     this.hasElopage = hasElopage
     this.deletedAt = user.deletedAt
     this.emailConfirmationSend = emailConfirmationSend
-    this.isAdmin = user.isAdmin
+    if (user.userRoles) {
```

Comment:
for sure, ...
but comparing with null is on my opinion more efficient than
instantiating an empty array, transfer over all layers this empty array to validate if the array is empty in case of no userRole...

Reply:
outdated